packages/generators/templates/shared/jest.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+	preset: '../../test/packages/jest-preset.js',
+	testEnvironment: 'jsdom',
+	testMatch: [ '<rootDir>/**/__tests__/**/*.[jt]s?(x)', '!**/.eslintrc.*' ],
+	setupFilesAfterEnv: [
+		'@testing-library/jest-dom/extend-expect',
+		'@automattic/calypso-build/jest/mocks/match-media',
```

Comment:
Do we really need this one? Seems like it's pretty specific and not necessary?

Reply:
Removed!